porch/func/wrapper-server/main.go

```diff
@@ -0,0 +1,133 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"net"
+	"os"
+	"os/exec"
+
+	pb "github.com/GoogleContainerTools/kpt/porch/func/evaluator"
+	"github.com/spf13/cobra"
```

Comment:
considering how simple the wrapper is, it may not be worth introducing cobra.

Reply:
`cobra` has something convenient that I use below like `cmd.ArgsLenAtDash()`.
But it's not something very complex, we can parse it ourselves.

Reply:
got it; your call. Since we are using some features that don't have equivalent, I'm happy to keep using cobra.